MonoZelda/Controllers/CollisionController.cs

```diff
@@ -91,7 +91,14 @@ private void HandleCollision(Collidable collidableA, Collidable collidableB, par
     private object[] GetMetadata(Collidable collidableA, Collidable collidableB)
     {
         // We need to implement this method to give back information about the collision
-        return null;
+        var metadata = new object[]
+        {
+            collidableA,
+            collidableB,
+            this,
+        };
```

Comment:
I know this is all temporary, but I am voicing that I don't think we should pass the collision controller to commands so we can keep coupling as loose as possible. I do understand that we can make an exception if it is needed.

Reply:
I agree, we would have to implemented some other way to remove collidables from the collisionController list, rather than passing the collisionController object to multiple classes